modules/python/init.zsh

```diff
@@ -116,38 +116,42 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) \
   # look for plugins of interest. Scanning shell '$path' isn't enough as they
   # can exist in 'pyenv' synthesized paths (e.g., '~/.pyenv/plugins') instead.
   local -a pyenv_plugins
+  local pyenv_virtualenvwrapper_plugin_found
```

Comment:
Testing revealed this line isn't necessary... but I was unclear if it's more idiomatic to declare the var since otherwise it's only set deep in nested `if`, but then tested at a higher level of scope.

Also unclear if when declaring it's best to declare it as a `local` or not... I don't understand why `pyenv_plugins` is declared as a `local` but `virtenv_sources` isn't...

Reply:
I think pyenv_plugins is declared locally so we can specify that it's an array. It seems reasonable to also declare pyenv_virtualenvwrapper_plugin_found up here.